src/main/java/com/google/devtools/build/lib/authandtls/credentialhelper/CredentialModule.java

```diff
@@ -47,11 +46,7 @@ public Iterable<Class<? extends OptionsBase>> getCommonCommandOptions() {
   public void beforeCommand(CommandEnvironment env) {
     // Update the cache expiration policy according to the command options.
     AuthAndTLSOptions authAndTlsOptions = env.getOptions().getOptions(AuthAndTLSOptions.class);
-    credentialCache
-        .policy()
-        .expireAfterWrite()
-        .get()
-        .setExpiresAfter(authAndTlsOptions.credentialHelperCacheTimeout);
+    cacheExpiry.setDefaultCacheDuration(authAndTlsOptions.credentialHelperCacheTimeout);
```

Comment:
Let's also amend the documentation for the `--credential_helper_cache_timeout` flag to make it clear that it only applies to credentials without an explicit expiry.

Reply:
Done